docs/Playing/turn-change.rst

```diff
@@ -0,0 +1,268 @@
+The Turn Change Sequence
+************************
+
+This page has been adapted from the legacy Freeciv `Wiki <https://freeciv.fandom.com/wiki/Turn>`_.
+
+Freeciv21 is a turn-based game. A turn is a period of the game when all players (human and AI) are allowed to
+make a specific number of game actions. Longturn online multi-player games have turns that are often set to a
+counter of 23 hours. This allows players all over the world an opportunity to log in and make thier moves at a
+reasonable time. Single player games (one human against the AI) do not have a counter set. The turn ends when
+all moves have been completed and then the human player will manually clicks the :guilabel:`End Turn` button.
+
+Turns are typically enumerated from ``T0``.
+
+The period of a turn when a specific player or player group can act in a game is called a *phase*. Most often
+Freeciv21 is played in the concurrent mode - there is only one phase per turn for everybody (AIs do thier
+moving only at phase beginning). But in the server options another mode is available: all players move in a
+queue, or all teams move in a queue. Most things happen to an empire either at the start or at the end of its
+phase (since the default mode is one-phase, these periods are commonly referred as "``turn start``" or
+"``turn end``", but in alternate playing they will happen to different players in different time) [#f1]_.
```

Comment:
I wouldn't use quote around "`fixed width text`". There are other occurrences thorough the text